docs/reference/interruptive-surveys.md

```diff
@@ -1,9 +1,11 @@
 ---
-id: interruptive-surveys
-title: Running Interruptive Surveys in FxA
-sidebar_label: Running Surveys in FxA
+title: Interruptive Surveys
 ---
 
+:::caution
+This feature is being removed in [FXA-3880](https://mozilla-hub.atlassian.net/browse/FXA-3880)
```

Comment:
I went ahead and specified in the ticket the documentation update is in ecosystem-platform